misc/install.sh

```diff
@@ -1,113 +1,9 @@
-cp "$HOME/screenly/misc/screenly.conf" "$HOME/.screenly/"
-
-echo "Enabling Watchdog..."
-sudo modprobe bcm2708_wdog > /dev/null
-sudo cp /etc/modules /etc/modules.bak
-sudo sed '$ i\bcm2708_wdog' -i /etc/modules
-sudo chkconfig watchdog on
-sudo cp /etc/watchdog.conf /etc/watchdog.conf.bak
-sudo sed -e 's/#watchdog-device/watchdog-device/g' -i /etc/watchdog.conf
-sudo /etc/init.d/watchdog start
-
-echo "Adding Screenly to autostart (via Supervisord)"
-sudo ln -s "$HOME/screenly/misc/supervisor" /etc/init.d/supervisor
-sudo ln -s "$HOME/screenly/misc/supervisord.conf" /etc/supervisor/supervisord.conf
-sudo ln -s "$HOME/screenly/misc/supervisord.conf" /etc/supervisord.conf
-sudo ln -s "$HOME/screenly/misc/supervisor_screenly.conf" /etc/supervisor/conf.d/screenly.conf
-sudo update-rc.d supervisor defaults
-sudo /etc/init.d/supervisor start > /dev/null
-
-echo "Making modifications to X..."
-[ -f "$HOME/.gtkrc-2.0" ] && rm -f "$HOME/.gtkrc-2.0"
-ln -s "$HOME/screenly/misc/gtkrc-2.0" "$HOME/.gtkrc-2.0"
-[ -f "$HOME/.config/openbox/lxde$SUFFIX-rc.xml" ] && \
-    mv "$HOME/.config/openbox/lxde$SUFFIX-rc.xml" "$HOME/.config/openbox/lxde$SUFFIX-rc.xml.bak"
-[ -d "$HOME/.config/openbox" ] || mkdir -p "$HOME/.config/openbox"
-ln -s "$HOME/screenly/misc/lxde-rc.xml" "$HOME/.config/openbox/lxde$SUFFIX-rc.xml"
-[ -f "$HOME/.config/lxpanel/LXDE$SUFFIX/panels/panel" ] && \
-    mv "$HOME/.config/lxpanel/LXDE$SUFFIX/panels/panel" "$HOME/.config/lxpanel/LXDE$SUFFIX/panels/panel.bak"
-
-# Cover both situations, as there have been traces of both in recent versions.
-[ -f "/etc/xdg/lxsession/LXDE/autostart" ] && \
-    sudo mv "/etc/xdg/lxsession/LXDE/autostart" "/etc/xdg/lxsession/LXDE/autostart.bak"
-[ -f "/etc/xdg/lxsession/LXDE$SUFFIX/autostart" ] && \
-    sudo mv "/etc/xdg/lxsession/LXDE$SUFFIX/autostart" "/etc/xdg/lxsession/LXDE$SUFFIX/autostart.bak"
-
-sudo sed -e 's/^#xserver-command=X$/xserver-command=X -nocursor/g' -i /etc/lightdm/lightdm.conf
-
-# Make sure we have proper framebuffer depth.
-if grep -q framebuffer_depth /boot/config.txt; then
-  sudo sed 's/^framebuffer_depth.*/framebuffer_depth=32/' -i /boot/config.txt
-else
-  echo 'framebuffer_depth=32' | sudo tee -a /boot/config.txt > /dev/null
-fi
-
-# Fix frame buffer bug
-if grep -q framebuffer_ignore_alpha /boot/config.txt; then
-  sudo sed 's/^framebuffer_ignore_alpha.*/framebuffer_ignore_alpha=1/' -i /boot/config.txt
-else
-  echo 'framebuffer_ignore_alpha=1' | sudo tee -a /boot/config.txt > /dev/null
-fi
-
-echo "Quiet the boot process..."
-sudo cp /boot/cmdline.txt /boot/cmdline.txt.bak
-sudo sed 's/$/ quiet/' -i /boot/cmdline.txt
-
-echo "Assuming no errors were encountered, go ahead and restart your computer."
+sudo apt-get update
+sudo apt-get install -y python-setuptools python-dev git-core
+sudo easy_install pip
+sudo pip install ansible
+
+ansible localhost -m git -a "repo=git://github.com/over64/screenly-ose.git dest=/home/pi/screenly version=master" || exit
```

Comment:
Interesting approach. However, I think this might make it more difficult to work with other branches. Perhaps we should just have a `git pull` above (before `pip install ...`).